plugins/ports-plugin/src/ports-plugin.ts

```diff
@@ -151,13 +161,24 @@ function onClosedPort(port: Port) {
 
 export async function start(context: theia.PluginContext): Promise<void> {
 
+    outputChannel = theia.window.createOutputChannel('Ports Plug-in');
+
     // first, grab ports of workspace
     const workspaceHandler = new WorkspaceHandler();
     workspacePorts = await workspaceHandler.getWorkspacePorts();
 
     redirectListeners = new Map<number, BusyPort>();
     redirectPorts = workspacePorts.filter(port => port.serverName.startsWith(SERVER_REDIRECT_PATTERN));
 
+    // initiate excluded ports
+    const excludedPortProperties: string[] = Object.keys(process.env).filter(key => key.startsWith(PORT_EXCLUDE_ENV_VAR_PREFIX));
+    excludedPortProperties.forEach(key => {
+        const value = process.env[key].toLocaleLowerCase() || '';
+        if (value !== 'no') {
```

Comment:
Sorry, why do we need this check?

Reply:
hello @AndrienkoAleksandr 

it's because you can do
```
export PORT_PLUGIN_EXCLUDE_3130=YES
```

but
```
export PORT_PLUGIN_EXCLUDE_3130=NO
```
will not exclude the port

Reply:
Oh, ok you want support ['yes', 'no'] values too. But in dockerfile you defined value in the boolean format:
> PORT_PLUGIN_EXCLUDE_3130=TRUE

How Do you want to cover false value?

Reply:
@AndrienkoAleksandr I added a fixup, is it ok for you ?

Reply:
OK, absolutely.